example/declarative.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+library grinder.example.declarative;
+
+import 'package:grinder/grinder.dart';
+
+const main = grind;
+
+@Task()
+init(GrinderContext context) {
+  context.log("I set things up");
+}
+
+@Task(depends: const ['init'])
+compile(GrinderContext context) {
+  context.log("I'm compiling now...");
+  //context.fail('woot');
+}
+
+@Task(depends: const ['compile'])
+deploy(GrinderContext context) {
+  context.log("deploying a");
+  context.log("deploying b");
+}
+
+@Task(name: 'default', depends: const ['deploy'])
```

Comment:
We haven't published the `default` change yet. What do you think about having a `default` bool property on a task, instead of depending on a naming convention? It would mean we wouldn't have to have a no-op method to hang an annotation off.